preprocessor/src/main.rs

```diff
@@ -34,17 +34,21 @@ struct Args {
     #[arg(long)]
     database_url: Option<String>,
 
-    /// The tile size which the reference image will be split into
-    #[arg(short, long, default_value_t = 1000)]
-    tile_size: u64,
-
     /// The number of CPU threads to use for processing arg
     #[arg(short, long, default_value_t = 1)]
     cpu_num: usize,
+
+    /// Calculate the amount of levels of detail
+    #[arg(long, default_value_t = false)]
```

Comment:
```suggestion
    #[arg(long)]
```
default values er redundant med boolske værdier